File: 01-Arrays/01-MinAndMax.java
/**
 * 01-MinAndMax
 */
class MinAndMax {

    public static void minAndMax(int arr[]) {

        int min = Integer.MAX_VALUE;
        int max = Integer.MIN_VALUE;

        for (int a : arr) {
            if (a < min) {
                min = a;
            }
            if (a > max) {
                max = a;
            }
        }
        System.out.println(min);
        System.out.println(max);
    }

    public static void main(String[] args) {

        int arr[] = { 2, 4, -3, 1, 3, -2, 1, -5, 7, 2 };
        minAndMax(arr);

    }

}

File: 01-Arrays/01-MinAndMax.kt
import java.util.LinkedList

fun minAndMax(list: List<Int>): List<Int> {

    var min = Int.MAX_VALUE;
    var max = Int.MIN_VALUE;

    val ans = mutableListOf<Int>()
    

    for (i in list) {
        if (i > max) {
            max = i;
        }
        if (i < min) {
            min = i;

        }
    }
    ans.add(min);
    ans.add(max);

    return ans;

}


fun main() {
    val arr = listOf(1, 2, 3, 4, 5, 6, 7, 88, 9);

    println(minAndMax(arr))
}
